File: FoodTruckApp/Domain/Models/FoodTruck.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FoodTruckApp.Domain.Models {
    public class FoodTruck {

        public int Id { get; set; }

        public string Name { get; set; }

        public string Image { get; set; }

        public string Description { get; set; }

        public IList<MenuItem> MenuItems { get; set; }

        public string Location { get; set; }

        public IList<Review> Reviews { get; set; }

        public ApplicationUser Owner { get; set; }
    }
}

File: FoodTruckApp/Presentation/ngApp/models/FoodTruck.ts
namespace FoodTruckApp.Models {

    export class FoodTruck {

        public id: number;

        public name: string;

        public image: string;

        public description: string;

        public menuItems: MenuItem[];

        public location: string;

        public reviews: Review[];

        public owner;

        public category: string;
    }
}